### PR TITLE
Hvc groups apis

### PR DESCRIPTION
**Pull request by sekharpanja**

### CODE REVIEW

File: mi/views/hvcgroup_views.py [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+
+    def _group_result(self, group):
+        """ Basic data about sector team - name & hvc's """
+        return {
+            'name': group.name,
+            'avg_time_to_confirm': self._get_avg_confirm_time(group),
+            'hvcs': self._make_targets(group.targets.all()),
+        }
+
+    def _get_hvc_wins_by_sector_hvc(self, group):
```
Comment (adammalinowski) [inline mi/views/hvcgroup_views.py]: Cant this be removed and replaced with `_get_group_wins`?

File: mi/views/hvcgroup_views.py [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+                self._get_current_financial_start_date(),
+                self._get_current_financial_end_date(),
+            ),
+        ).exclude(
+            hvc=''
+        ).exclude(
+            hvc__isnull=True
+        ).select_related('confirmation')
+
+    def _breakdowns(self, wins):
```
Comment (adammalinowski) [inline mi/views/hvcgroup_views.py]: Are this method and `_breakdowns_cumulative` the same as those methods on the Base class? If so they we don't need them here too.

File: mi/views/hvcgroup_views.py [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+                'number': {
+                    'confirmed': self.non_export_confirm_cu_number,
+                    'non_confirmed': self.non_export_non_cu_number,
+                    'total': (self.non_export_confirm_cu_number + self.non_export_non_cu_number),
+                }
+            }
+        }
+
+
+class HVCGroupsListView(BaseHVCGroupMIView):
```
Comment (adammalinowski) [inline mi/views/hvcgroup_views.py]: Let's try to start documenting all our classes!

File: mi/views/hvcgroup_views.py (hunk 2 above)
Comment (sekharpanja) [inline mi/views/hvcgroup_views.py]: Subtly different, it doesn't have non_hvc in it. I can refactor both to somehow manage it in one place.

Comment (adammalinowski) [inline mi/views/hvcgroup_views.py]: Ah OK. I think I may be touching the base method in may branch fixing sector calculations, so lets leave this for now.

**Pull request merged**